squad/plugins/linux_log_parser.py

```diff
@@ -4,26 +4,28 @@
 from collections import defaultdict
 from squad.plugins import Plugin as BasePlugin
 from squad.core.models import SuiteMetadata
+from django.template.defaultfilters import slugify
 
 
 logger = logging.getLogger()
 
 REGEX_NAME = 0
 REGEX_BODY = 1
+REGEX_EXTRACT_NAME = 2
 
 MULTILINERS = [
-    ('check-kernel-exception', r'-+\[? cut here \]?-+.*?-+\[? end trace \w* \]?-+'),
-    ('check-kernel-kasan', r'=+\n\[[\s\.\d]+\]\s+BUG: KASAN:.*?=+'),
-    ('check-kernel-kfence', r'=+\n\[[\s\.\d]+\]\s+BUG: KFENCE:.*?=+'),
+    ('check-kernel-exception', r'-+\[? cut here \]?-+.*?-+\[? end trace \w* \]?-+', r"\d][^\+\n]*"),
```

Comment:
I saw you added some text to other patterns, like `BUG: KASAN:`, `Unhandled` etc, but for check-kernel-exception there's no text.

Reply:
Thanks! I didn't add it to this case since I didn't think including "cut here" in the name was very useful. There was nothing else guaranteed to always be in the extracted log line for this case.

Reply:
humm, makes sense. I think the real test name will come from its contents anyways. All good!